Touch Bar Simulator/AppDelegate.swift

```diff
@@ -17,6 +19,28 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 		$0.button!.toolTip = "Right-click or option-click for menu"
 	}
 
+	struct KeyboardShortcutRecorder: SwiftUI.View {
+		var title: String
+		var shortcut: KeyboardShortcuts.Name
+		var body: some View {
+			HStack {
+				Text(title)
+				KeyboardShortcuts.Recorder(for: shortcut)
+			}
+		}
+	}
+
+	lazy var keyboardShortcutsVC = NSHostingController(rootView:
+		AnyView(
```

Comment:
Instead of `KeyboardShortcutRecorder`, I would extract all the SwiftUI code into a view.

Reply:
Also, don't use `AnyView`.

Reply:
I'm still a noob at SwiftUI and the compiler was yelling at me for letting a `some View` propagate into the generic parameter of `NSHostingController`, so I saw the suggestion to use `AnyView`. Is there a better way to deal with this that you know of?

Reply:
The easiest is to just not store the View. `some` is currently very limited and doesn't work everywhere.

Reply:
😕

<img width="606" alt="Screen Shot 2020-05-25 at 11 37 21 PM" src="https://user-images.githubusercontent.com/16456186/82858026-b719b680-9ee0-11ea-8680-af00db24290b.png">

Reply:
Also, sorry, what do you mean by this? Extract into an `NSView`?

> Instead of `KeyboardShortcutRecorder`, I would extract all the SwiftUI code into a view.

Reply:
Oh never mind, I think I get it, disregard my last two comments 😉